### PR TITLE
TC_07.002.01 | People > Sort People > Sort by UserID

### PR DESCRIPTION
**Pull request by annaUniversal**
https://trello.com/c/fQzAqKN2/496-tc07002-people-sort-people

### CODE REVIEW

**Review by AlyonkaNY — APPROVED** [review]

**Review by SerHemb — APPROVED** [review]

**Review by AlyonkaNY — APPROVED** [review]

**Comment by annaUniversal** [comment]
[TC_07.002.01 | People > Sort People > Sort by UserID](https://trello.com/c/fQzAqKN2/496-tc0700201-people-sort-people-sort-by-userid)

**Comment by annaUniversal** [comment]
[TC_07.002.01 | People > Sort People > Sort by UserID](https://trello.com/c/fQzAqKN2/496-tc0700201-people-sort-people-sort-by-userid)

File: pom.xml [hunk 1]
```diff
@@ -27,5 +27,11 @@
             <artifactId>selenium-java</artifactId>
             <version>4.19.1</version>
         </dependency>
+
+        <dependency>
+            <groupId>com.github.javafaker</groupId>
+            <artifactId>javafaker</artifactId>
+            <version>1.0.2</version>
+        </dependency>
```
Comment (HelenMyronyuk) [inline pom.xml:35]: изменения пома нужно апрувить у Сергея

**Comment by aqaestuser** [comment]
не прошел CI и забросили его - закрываю

**Pull request closed without merging**